### ISSUE TITLE
Pip Install Issue with python 3.4.2

### ISSUE THREAD

**Issue by nuarhu** [eventials/django-pjax#8]
Hi!

I'm trying to install django-pjax in a virtual env running with python 3.4.2 (the highest available for debian). The installation fails with the following error:

```
$ pip install django-pjax
Collecting django-pjax
  Using cached django-pjax-1.4.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-13670p7i/django-pjax/setup.py", line 17, in <module>
        long_description=read('README.rst'),
      File "/tmp/pip-build-13670p7i/django-pjax/setup.py", line 10, in read
        return open(os.path.join(os.path.dirname(__file__), fname)).read()
      File "/usr/lib/python3.4/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 148: ordinal not in range(128)
```

The same error happens when installing the master:

```
$ pip install git+git://github.com/eventials/django-pjax.git
```

The problem is that the README.rst is UTF-8.

Thanks!

Note: There is no problem installing this in 3.4.3 or 3.5.

**Comment by ReZeroS** [comment]
Have you solved this problem? My problem seems like yours:

```
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\AppData\Local\Temp\pip-build-1169aa2q\django-pjax\setup.py", line 17, in <module>
        long_description=read('README.rst'),
      File "C:\Users\AppData\Local\Temp\pip-build-1169aa2q\django-pjax\setup.py", line 10, in read
        return open(os.path.join(os.path.dirname(__file__), fname)).read()
    UnicodeDecodeError: 'gbk' codec can't decode byte 0x9d in position 232: illegal multibyte sequence

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in C:\Users\AppData\Local\Temp\pip-build-1169aa2q\django-pjax\
```

And the version is 3.6.2, by the way, it woks well on py2.7

**Comment by nuarhu** [comment]
We are not using this anymore. I've implemented it directly.

**Comment by Almenon** [comment]
In windows 10 Home w/ python 3.6.0 I get:

> Collecting django-pjax==1.4 (from -r requirements\common.pip (line 35))
  Using cached https://files.pythonhosted.org/packages/df/08/f662091f67609087f7f6c8b98e3b344279b3d01fd458d655a858193d9b01/django-pjax-1.4.tar.gz
    Complete output from command python setup.py egg_info:

```python
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\almenon\AppData\Local\Temp\pip-install-9eil03a0\django-pjax\setup.py", line 17, in <module>
        long_description=read('README.rst'),
      File "C:\Users\almenon\AppData\Local\Temp\pip-install-9eil03a0\django-pjax\setup.py", line 10, in read
        return open(os.path.join(os.path.dirname(__file__), fname)).read()
      File "f:\envw\lib\encodings\cp1252.py", line 23, in decode
        return codecs.charmap_decode(input,self.errors,decoding_table)[0]
    UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 232: character maps to <undefined>
```

> Command "python setup.py egg_info" failed with error code 1 in C:\Users\almenon\AppData\Local\Temp\pip-install-9eil03a0\django-pjax\

**Comment by caleb15** [comment]
We just ran into this error with python 3.6.8 on Ubuntu 18. #11 should fix it.

**Comment by Mugicha** [comment]
I also get same error with python3.7.3 on windows10.
But I solved this by editing as bellow.
In read(fname) in setting.py, add encoding='utf8' in open function.

**Comment by smyja** [comment]
> I also get same error with python3.7.3 on windows10.
> But I solved this by editing as bellow.
> In read(fname) in setting.py, add encoding='utf8' in open function.

Which setting.py? Do you have the modified repo you used?

**Comment by skrater** [comment]
Fixed by #11.
